### PR TITLE
Update README with some links to airgun/nailgun/ssh2-python

### PR DESCRIPTION
**Pull request by mshriver**
Using this to test PRT trigger, these updates are largely inconsequential but might be nice to merge anyway.

### CODE REVIEW

**Comment by mshriver** [comment]
test-robottelo

**Comment by mshriver** [comment]
test-robottelo

**Review by JacobCallahan — APPROVED** [review]

**Comment by mshriver** [comment]
trigger: test-robottelo

File: README.rst [hunk 1]
```diff
@@ -24,9 +24,9 @@ Robottelo
 automated, suited for use in a continuous integration environment, and `data
 driven`_. There are three types of tests:
 
-* UI tests, which rely on Selenium's `WebDriver`_.
-* CLI tests, which rely on `Paramiko`_.
-* API tests, which rely on `Requests`_.
+* UI tests, which rely on Selenium's `WebDriver`_ through `airgun`_.
+* CLI tests, which rely on `Paramiko`_ - and are being converted to using `ssh2-python`_.
```
Comment (swadeley) [inline README.rst:28]:
```suggestion
* CLI tests, which currently rely on `Paramiko`_ , but are being converted to use `ssh2-python`_.
```
Please avoid dashes and slashes in text

**Review by swadeley — APPROVED** [review]
ACK

**Pull request merged**